### ISSUE TITLE
The shape categories visible by default in the web editor are not visible by default in the extension

### ISSUE THREAD

**Issue by phiresky** [hediet/vscode-drawio#198]
When creating a new empty diagram on diagrams.net, it looks like this:

![image](https://user-images.githubusercontent.com/2303841/105173655-3ebfcf80-5b19-11eb-9bbe-a1b62c610fec.png)

In your extension, it instead looks like this:

![image](https://user-images.githubusercontent.com/2303841/105173701-4ed7af00-5b19-11eb-845b-d44b4d08fdee.png)

It would be helpful if the defaults would match, especially the "Basic" and "Arrows" categories are disabled:

![image](https://user-images.githubusercontent.com/2303841/105173794-6e6ed780-5b19-11eb-9591-b0ec63276f88.png)

**Comment by hediet** [comment]
Did you try this on a clean installation?
Otherwise, can you reset the drawio settings in vscode and see how it looks like?

**Comment by phiresky** [comment]
Yep, I just installed the extension on a computer I did not have it installed before. It also happened the last time i installed it on a different computer.

If I reset the settings by removing hediet.vscode-drawio.local-storage and .theme they are gone again

**Comment by hediet** [comment]
I actually intentionally only show "general", but you can change it and your change should be persisted.

If this issue gets 5 upvotes, I will change the default.
